Packs/PAN-OS/Integrations/Panorama/README.md

```diff
@@ -5684,7 +5684,7 @@ Gets information from all PAN-OS systems in the topology.
 
 ### pan-os-platform-get-device-groups
 ***
-Gets the operational information of the device groups in the topology.
+Gets the operational information of the device groups in the topology(only device groups with associated devices will be listed by this command).
```

Comment:
Please update this in the description of the yml as well.

Reply:
I've added the description to the yml as well